worlds/hk/docs/setup_pt_br.md

```diff
@@ -0,0 +1,52 @@
+# Guia de configuração para Hollow Knight no Archipelago
+
+## Programas obrigatórios
+* Baixe e extraia o Lumafly Mod Manager(gerenciador de mods Lumafly) do [Site Lumafly](https://themulhima.github.io/Lumafly/).
+* Uma cópia legal de Hollow Knight.
+   * Versões Steam, Gog, e Xbox Game Pass do jogo são suportadas.
+   * Windows, Mac, e Linux (incluindo Steam Deck) são suportados.
+
+## Instalando o mod Archipelago Mod usando Lumafly
+1. Abra o Lumafly e confirme que ele localizou sua pasta de instalação do hollow knight.
```

Comment:
I would capitalize the initials of Hollow Knight here, unless you're talking of a specific folder, which doesn't seem to be the case from me following the tutorial.